process/transactions/scrsDataToTransactions.go

```diff
@@ -107,6 +114,26 @@ func hasSuccessfulSCRs(tx *data.Transaction) bool {
 	return false
 }
 
+func hasCrossShardPendingTransfer(tx *data.Transaction) bool {
+	for _, scr := range tx.SmartContractResults {
+		splitData := strings.Split(string(scr.Data), atSeparator)
+		if len(splitData) < 2 {
+			return false
```

Comment:
is it ok to return here? this could be the first element in a slice of 100 SCRs.. is it ok to return after the first scr? same as above

Reply:
no. good catch refactored.